modyn/supervisor/internal/triggers/trigger_datasets/fixed_keys_dataset.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import math
+from typing import Callable, Generator, Iterable, Optional
+
+import grpc
+from modyn.storage.internal.grpc.generated.storage_pb2 import (  # pylint: disable=no-name-in-module
+    GetRequest,
+    GetResponse,
+)
+from modyn.storage.internal.grpc.generated.storage_pb2_grpc import StorageStub
+from modyn.utils import (
+    BYTES_PARSER_FUNC_NAME,
+    MAX_MESSAGE_SIZE,
+    deserialize_function,
+    grpc_connection_established,
+    instantiate_class,
+)
+from torch.utils.data import IterableDataset, get_worker_info
+from torchvision import transforms
+
+logger = logging.getLogger(__name__)
+
+
+# TODO(#275): inherit common abstraction of dataset
+class FixedKeysDataset(IterableDataset):
```

Comment:
Can you add a docstring here what this does and why we need it? For the sampling dataset, I remember, but here some docs would be great. Thank you!

Reply:
done.